nextflow.config

```diff
@@ -279,14 +279,14 @@ manifest {
     description     = """A bioinformatics best-practice analysis pipeline for calling structural variants (SVs), copy number variants (CNVs) and repeat region expansions (RREs) from short DNA reads."""
     mainScript      = 'main.nf'
     defaultBranch   = 'main'
-    nextflowVersion = '!>=24.04.2'
+    nextflowVersion = '!>=24.10.0'
     version         = '0.3.0dev'
     doi             = ''
 }
 
 // Nextflow plugins
 plugins {
-    id 'nf-schema@2.3.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
+    id 'nf-schema@2.2.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
```

Comment:
is this a downgrade or was 2.3.0 a typo?

Reply:
There's a bug in v2.3.0 when using the workflow output definitions. Using v2.2.0 is fine for now, I'll update it once v2.4.0 has been released (this contains that fix)